Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: build-garage-headless build-garage-nvidia \
+	run-garage-headless run-garage-nvidia
+
+build-garage-headless: docker/Dockerfile check-mjkey
+	@ docker build \
+		-f docker/Dockerfile \
+		--build-arg MJKEY="$(MJKEY)" \
+		--build-arg IMAGE_TYPE="headless" \
+		-t rlworkgroup/garage-headless .
+
+build-garage-nvidia: docker/Dockerfile check-mjkey
+	@ docker build \
+		-f docker/Dockerfile \
+		--build-arg MJKEY="$(MJKEY)" \
+		--build-arg IMAGE_TYPE="nvidia" \
+		--build-arg PARENT_IMAGE="nvidia/opengl:1.0-glvnd-runtime-ubuntu16.04" \
+		-t rlworkgroup/garage-nvidia .
+
+run-garage-headless: build-garage-headless
+	@ docker run \
+		-it \
+		--rm \
+		-v $(shell pwd)/data:/root/code/garage/data \
+		-e MJKEY="$(MJKEY)" \
+		rlworkgroup/garage-headless $(RUN_CMD)
+
+run-garage-nvidia: build-garage-nvidia
+	xhost +local:docker
+	@ docker run \
+		-it \
+		--rm \
+		--runtime=nvidia \
+		-v /tmp/.X11-unix:/tmp/.X11-unix \
+		-v $(shell pwd)/data:/root/code/garage/data \
+		-e DISPLAY=$(DISPLAY) \
+		-e QT_X11_NO_MITSHM=1 \
+		-e MJKEY="$(MJKEY)" \
```

Comment:
suggestion:
if the environment variable MJKEY is empty, attempt to read it from the disk (~/.mujoco/mjkey.txt) into the environment variable and produce a message saying you automatically detected the key at that location.

Reply:
Done